cli/args/mod.rs

```diff
@@ -812,6 +812,23 @@ impl CliOptions {
       }
     }
 
+    // discourage using --allow-run without an allow list
+    let has_empty_allow_run_flag = flags
+      .permissions
+      .allow_run
+      .as_ref()
+      .map(|r| r.is_empty())
+      .unwrap_or(false);
+    if has_empty_allow_run_flag
+      && !flags.permissions.allow_all
+      && flags.permissions.deny_run.is_none()
+    {
+      log::warn!(
+        "{} --allow-run without an allow list is insecure",
```

Comment:
we should link to docs

Reply:
I linked to the docs in this code and then updated them here: https://github.com/denoland/docs/pull/750/files

Reply:
I think the tone of the warning message, `--allow-run without an allow list is insecure` might be overly strong as it may lead to the wrong impression that using an allow list is guaranteed yo be secure. 

I suggest considering a more nuanced warning, such as:
`--allow-run without an allow list increases security risks. Consider using an allow list and carefully managing its contents`
This emphasizes the potential benefits of using an allow list while acknowledging its limitations

Reply:
I updated the message